cg-spec/editors_draft.html

```diff
@@ -301,7 +301,7 @@ <h2>Background and Motivation</h2>
     <section class="informative">
       <h2>Overview</h2>
 
-<p>This specification describes RDF-star, an extension of RDF's conceptual data model and concrete syntaxes, providing a more compact form of reification. This model and syntaxes enable the creation of concise triples that reference other triples as subject and object resources.
+<p>This specification describes RDF-star, an extension of RDF's conceptual data model and concrete syntaxes, providing a compact alternative to <a href="https://www.w3.org/TR/rdf11-mt/#reification">standard reification</a>. This model and syntaxes enable the creation of concise triples that reference other triples as subject and object resources.
```

Comment:
@pchampin -- Sorry I didn't catch this before merge... I can make a PR, if you prefer.
```suggestion
<p>This specification describes RDF-star, an extension of RDF's conceptual data model and concrete syntaxes, which provides a compact alternative to <a href="https://www.w3.org/TR/rdf11-mt/#reification">standard RDF reification</a>. This model and its syntaxes enable the creation of concise triples that reference other triples as subject and object resources.
```

Reply:
No problem. I can apply it directly on `master`.
Question though: I don't really like "this model and *its* syntaxes" because the idea is to refer to everything that was mentioned in the previous sentence, i.e. the model AND the syntaxes. I understand that the singular "this" is a problem, but would "these model and syntaxes" work? Or "this model and these syntaxes"?

Reply:
I'd be OK with "this model and these syntaxes", though to me it implies that no other syntaxes can "enable ... concise ... resources."

To my mind, "This model" and _any syntax that expresses it_ (which includes but is not limited to those we're currently specifying) can "enable ... concise ... resources." hence, "its syntaxes".

Reply:
I get your point. "its syntax" it is, then :)
Done in 7ff7179